src/test/system_tests/ceph_s3_tests/s3-tests-lists/nsfs_s3_tests_black_list.txt

```diff
@@ -363,9 +363,6 @@ s3tests_boto3/functional/test_s3.py::test_lifecycle_expiration_tags2
 s3tests_boto3/functional/test_s3.py::test_lifecycle_expiration_versioned_tags2
```

Comment:
I think it would be good if we will add our own tests as well. check that it works on both containerized and non-containerized environments

Reply:
Sure.